### ISSUE TITLE
Datasource needs to support changing the caption which is not currently supported.

### ISSUE THREAD

**Issue by benlower** [tableau/document-api-python#98]
I'm changing a workbook's datasource and then republishing to Server. However, the Data Source Name in Vizportal shows the old name. I've tried changing the datasource.caption to no avail. The only other place the ds name shows up in my workbook's XML is in worksheets.table.view.datasources:

```
<table>
        <view>
          <datasources>
            <datasource caption='<OLD CAPTION DATASOURCE>' name='<DS NAME>' />```

**Comment by RussTheAerialist** [comment]
Looking into the code, we don't currently support changing the caption, so that's something we'll need to add.

**Comment by RussTheAerialist** [comment]
Fixed by #99
